drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -357,6 +357,11 @@ void RasterizerGLES2::set_boot_image(const Ref<Image> &p_image, const Color &p_c
 	}
 	glClear(GL_COLOR_BUFFER_BIT);
 
+	if (p_image.is_null()) {
```

Comment:
That's hacky, it would be better to use a dedicated method to set the clear color when no image is required.

Reply:
Added `set_boot_color(color)` then.